README.md

```diff
@@ -7,7 +7,11 @@
 
 3. `npm install`
 
-4. start bodhi-graphql at 5555
+4. Download corresponding qtum from https://github.com/qtumproject/qtum/releases, unzip it,change the folder name to `mac` or `win32` or `win64` or `linux64` or `linux32`, and put it inside `./qtum` folder, if `qtum` folder doesn't exist, create one. 
```

Comment:
this is wrong
1. `./qtum` should exist
2. mac/win32/win64/linux64/linux32 folder structure should remain
3. for dev, only mac version is needed